hrqb/tasks/sql/employees.sql

```diff
@@ -3,8 +3,31 @@ Query for Employee data.
 
 CHANGELOG
     - 2024-05-13 Query created and added
+    - 2024-07-09 Added CTEs to gather last appointment transaction, to get termination
+        or retirement reason
 */
 
+with ordered_appt_txns as (
+    select
+        a.MIT_ID,
+        at.HR_PERSONNEL_ACTION,
+        at.HR_ACTION_REASON,
+        row_number() over (
+            partition by a.MIT_ID
+            order by a.APPT_TX_BEGIN_DATE desc, a.APPT_TX_END_DATE desc
+        ) as tx_row_num
```

Comment:
But if you are going to keep using `tx*`, can it be consistently `txn` or `tx`?

Reply:
Good call: updated to `txn` everywhere.  In retrospect, those CTE tables should have been singularly named anyhow.